### DISCUSSION TITLE
Rule for mqtt sensor

### DISCUSSION THREAD

**Discussion by pistoletov1974** [arendst/Tasmota#21035] [Q&A] [answered]
how can I publish the values of all sensors based on a some trigger  in a rule?
like this rule:
ON POWER1#State == ON DO publish cmnd/tasmota/Status 0 ENDON

`
12:44:51.635 MQT: stat/tasmota_don1/RESULT = {"POWER1":"ON"}
12:44:51.639 MQT: stat/tasmota_don1/POWER1 = ON
12:44:51.656 RUL: POWER1#STATE == ON performs "publish cmnd/tasmota/Status 0"
12:44:51.664 MQT: cmnd/tasmota/Status = 0

**Comment by sfromis** [comment, marked as answer]
If what you want is `Status 0`, do not use the `Publish` command at all. Executing the `Status0` command (note no space, `Status0` instead of `Status 0`), and the result will be published. If what you want is only sensors, using `Status 10` (with a space now) gives you a shorter payload than the full `Status 0`.
